app/assets/javascripts/discourse/app/components/topic-list.js

```diff
@@ -170,14 +170,13 @@ export default Component.extend(LoadMore, {
   },
 
   click(e) {
-    let self = this;
     let onClick = function (sel, callback) {
```

Comment:
You don't need a `bind(this)` if you use the fat arrow `(sel, callback) =>` syntax which we prefer.